iothub/service/src/Common/Security/ISharedAccessSignatureCredential.cs

```diff
@@ -6,17 +6,38 @@ namespace Microsoft.Azure.Devices.Common.Security
     using System;
 
     using Microsoft.Azure.Devices.Common.Data;
-      
+
+    /// <summary>
+    /// Credential interface used for authentication and authorization
+    /// </summary>
     public interface ISharedAccessSignatureCredential
     {
+        /// <summary>
+        /// Indicates if the token has expired
+        /// </summary>
         bool IsExpired();
```

Comment:
q - in .NET, do all methods in a public interface have public visibility? should we declare it as such explicitly?

Reply:
according to https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers#other-types


"Interface members are public by default because the purpose of an interface is to enable other types to access a class or struct."